test/src/xquery/maps/mapsLookup.xql

```diff
@@ -0,0 +1,23 @@
+xquery version "1.0";
```

Comment:
@joewiz, you are right. It's simple to fix by that [code](https://github.com/eXist-db/exist/blob/005045fda2ee0df0c3871b12d1b90a79f086ffbc/src/org/exist/xquery/ConcatExpr.java#L14-L17). That going to be staff for next PR.